src/librustc_typeck/check/intrinsic.rs

```diff
@@ -67,7 +67,7 @@ pub fn intrisic_operation_unsafety(intrinsic: &str) -> hir::Unsafety {
     match intrinsic {
         "size_of" | "min_align_of" | "needs_drop" |
         "add_with_overflow" | "sub_with_overflow" | "mul_with_overflow" |
```

Comment:
should these be changed to match the others, or is their different signature sufficient to motivate a different naming convention? (this naming convention is not reflected in the std names)

Reply:
These match the LLVM intrinsics. We could rename them later but they're not *outright wrong* like `overflowing_*` intrinsics were.